persistence/sql/migratest/fixtures/networks/00000000-0000-0000-0000-000000000000.json

```diff
@@ -0,0 +1,3 @@
+{
+  "id": "00000000-0000-0000-0000-000000000000"
```

Comment:
There is an assertion in the test to make sure the NID is not zero. We can't have it in the fixture because it is randomly generated by the migration that is being tested.

Reply:
Ok, I have looked into the test and can confirm that. I've removed the fixture and the line calling it, because I think it adds little value to the test since we just override the nid with uuid.Nil